app/src/main/java/com/github/HumanLearning2021/HumanLearningApp/room/RoomCategorizedPicture.kt

```diff
@@ -0,0 +1,12 @@
+package com.github.HumanLearning2021.HumanLearningApp.room
+
+import androidx.room.Embedded
+import androidx.room.Relation
+
+data class RoomCategorizedPicture(
+    @Embedded val category: RoomCategory,
+    @Relation(
+        parentColumn = "categoryId",
+        entityColumn = "categoryId",
+    ) val picture: RoomPicture
```

Comment:
why isn't this simply a "url" field?

Reply:
To make the translation RoomCategorizedPicture to (Firestore)CategorizedPicture easier

Reply:
I dont get how but we can talk about it later on

Reply:
After reflection this will be change to Uri